documentation/Dockerfile

```diff
@@ -0,0 +1,40 @@
+## Base ########################################################################
+# Use a larger node image to do the build for native deps (e.g., gcc, python)
+FROM node:lts as base
+
+# Reduce npm log spam and colour during install within Docker
+ENV NPM_CONFIG_LOGLEVEL=warn
+ENV NPM_CONFIG_COLOR=false
+
+# We'll run the app as the `node` user, so put it in their home directory
+WORKDIR /home/node/app
+# Copy the source code over
+COPY --chown=node:node . /home/node/app/
+
+## Development #################################################################
+# Define a development target that installs devDeps and runs in dev mode
+FROM base as development
+WORKDIR /home/node/app
+# Install (not ci) with dependencies, and for Linux vs. Linux Musl (which we use for -alpine)
+RUN npm install
+# Switch to the node user vs. root
+USER node
+# Expose port 3000
+EXPOSE 3000
+# Start the app in debug mode so we can attach the debugger
+CMD ["npm", "start"]
```

Comment:
I think we don't need it for the dev environment. On dev, the developer can start the server easily on the local machine